PygmalionAI/aphrodite-engine
# feat: add grafana for metrics

To test:

1. Launch Aphrodite OpenAI server (launch with `--disable-log-requests` to reduce spam).
2. Navigate to `examples/monitoring` directory and run `docker compose up`.
3. Send requests to the server.
4. Navigate to http://localhost:2242/metrics for raw Prometheus.
5. Navigate to http://localhost:3000 for Grafana. Default username and password are both `admin`.
6. Navigate to http://localhost:3000/connections/datasources/new and Select Prometheus.
7. Scroll down, add type `http://prometheus:9090` for the Prometheus URL. Click `Save & Test`.
8. Navigate to http://localhost:3000/dashboard/import. 
9. Either upload `grafana.json` or type `20397` for the dashboard URL.
10. Stats should begin appearing.
